mk/k3d.mk

```diff
@@ -113,9 +113,13 @@ k3d/configure/metallb:
 
 .PHONY: k3d/wait
 k3d/wait:
-	until \
```

Comment:
The previous script will wait infinitely if the conditions will never met, which cause CI jobs never complete and we don't have a change to detect the detailed errors.